Documentation/INSTALL.md

```diff
@@ -2,49 +2,48 @@
 
-Serenity currently has no support for USB but some machines will emulate PS/2 keyboards and mice in the BIOS via USB. BIOS USB PS/2 emulation can be buggy so having real PS/2 input devices is recommended. A minimum of 128 MB RAM and a Pentium III class CPU are required.
+Serenity currently has no support for USB, but some machines will emulate PS/2 keyboards and mice in the BIOS via USB. BIOS USB PS/2 emulation can be buggy, so having real PS/2 input devices is recommended. A minimum of 128 MB RAM and a Pentium III class CPU is required.
 
-At present there is no real GPU support so don't expect OpenGL, Vulkan nor accelerated video playback and encoding support. Serenity currently relies upon VESA BIOS extensions to provide its display output and so it only runs on BIOS-based PCs. There is no WiFi support and the only physical network card chipset currently supported is the RTL8139. The sole sound card supported is the SoundBlaster 16 ISA.
+There is no real GPU support, so don't expect OpenGL, Vulkan, or accelerated video playback and encoding support. Serenity only runs on BIOS-based PCs because of reliance on VESA BIOS extensions to provide its display output. There is no WiFi support and the only physical network card chipset currently supported is the RTL8139. The sole sound card supported is the SoundBlaster 16 ISA.
 
-For more details on known working hardware see the [SerenityOS Hardware Compatibility List](https://github.com/SerenityOS/serenity/blob/master/Documentation/HardwareCompatibility.md).
+See the [SerenityOS Hardware Compatibility List](https://github.com/SerenityOS/serenity/blob/master/Documentation/HardwareCompatibility.md).
 
 ## Creating a Serenity GRUB disk image
 
-Before creating a Serenity disk image, you need to build the OS as described in the [SerenityOS build instructions](https://github.com/SerenityOS/serenity/blob/master/Documentation/BuildInstructions.md). Follow those instructions up to and including running **make install**. After the OS has built, run **make grub-image** to create a new file called **grub_disk_image** with GRUB2 installed that can be booted on a real PC.
+Before creating a Serenity disk image, you need to build the OS described in the [SerenityOS build instructions](https://github.com/SerenityOS/serenity/blob/master/Documentation/BuildInstructions.md). Follow those instructions up to and including running **make install**. After the OS has been built, run **make grub-image** to create a new file called **grub_disk_image** with GRUB2 installed to be booted on a real PC.
 
 The final step is copying **grub_disk_image** onto the disk you wish to use to boot Serenity using a command such as:
 
-```
+```bash
 $ sudo dd if=grub_disk_image of=/dev/sdx bs=64M && sync
 ```
```

Comment:
That's not correct, this line has a leading `$` symbol to indicate a shell session.

Reply:
Maybe the behavior changed, but the code fence seems to apply syntax highlighting regardless of the leading `$`.

![image](https://user-images.githubusercontent.com/10052309/104819739-2c7e2280-57f5-11eb-96af-ad83f1b06101.png)